face-finder-api/Chart.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: v2
+name: face-finder-api
+description: A Helm chart for face-finder-api
+version: v1.0.0
```

```suggestion
version: v0.1.0
```

We should start at the 1st unstable version